File: python/4th_week/contohsoal2/TokoBangunan.py
class TokoBangunan:

    def __init__(self, kodeToko=0, namaToko="", alamatToko=""):
        self.kodeToko = kodeToko
        self.namaToko = namaToko
        self.alamatToko = alamatToko
    
    def getKodeToko(self):
        return self.kodeToko
    
    def getNamaToko(self):
        return self.namaToko

    def getAlamatToko(self):
        return self.alamatToko
    
    def setKodeToko(self, kodeToko):
        self.kodeToko = kodeToko
    
    def setNamaToko(self, namaToko):
        self.namaToko = namaToko
    
    def setAlamatToko(self, alamatToko):
        self.alamatToko = alamatToko
    

File: cpp/4th_week/contohsoal2/TokoBangunan.cpp
#include <iostream>
#include <string>

using namespace std;

class TokoBangunan
{
private:
    /* data */
    string kodeToko, namaToko, alamatToko;
public:
    TokoBangunan() {}
    TokoBangunan(string kodeToko, string namaToko, string alamatToko){
        this->kodeToko = kodeToko;
        this->namaToko = namaToko;
        this->alamatToko = alamatToko;
    }

    //  get and set kodetoko
    string getKodeToko() {
        return kodeToko;
    }

    void setKodeToko(string kodeToko) {
        this->kodeToko = kodeToko;
    }

//    get and set nama toko
    string getNamaToko() {
        return namaToko;
    }

    void setNamaToko(string namaToko) {
        this->namaToko = namaToko;
    }

//    get and set alamat toko
    string getAlamatToko() {
        return alamatToko;
    }

    void setAlamatToko(string alamatToko) {
        this->alamatToko = alamatToko;
    }
    
    ~TokoBangunan() {}
};